report_qweb_txt/demo/res_users_report.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+<template id="report_res_users_csv">name;login;last_login_date;lang
+<t t-foreach="docs" t-as="o"><t t-esc="o.name"/>;<t t-esc="o.login"/>;<t t-esc="o.login_date"/>;<t t-esc="o.lang"/>
```

Comment:
@gdgellatly 'csv' in France defaults to using semi colons (because comma is used as decimal separator). But I agree. 
@alexis-via I suggest using config.csv_internal_sep for this (this is a parameter which can be set in the Odoo configuration file).